src/generator/protocol/operations.ts

```diff
@@ -148,6 +154,89 @@ function formatParamValue(param: Parameter, imports: ImportManager): string {
   }
 }
 
+// use this to generate the code that will help process values returned in response headers
+function formatHeaderResponseValue(header: LanguageHeader, imports: ImportManager): HeaderResponse {
+  let headerText = <HeaderResponse>{};
+  let text = ``;
+  switch (header.schema.type) {
+    case SchemaType.Boolean:
+      imports.add('strconv');
+      text = `\tval, err := strconv.ParseBool(resp.Header.Get("${header.header}"))\n`;
+      text += `\tif err != nil {\n`;
+      text += `\t\treturn nil, err\n`;
+      text += `\t}\n`;
+      headerText.body = text;
+      headerText.respObj = `{RawResponse: resp.Response, ${header.name}: &val}`;
```

Comment:
The "RawResponse: resp.Response" part is declared by the caller.  I suggest making this part a parameter to this function so it's only defined once by the caller.

Reply:
Ok added it and passed it into the function as a param